src/prelude/string.savi

```diff
@@ -440,6 +442,12 @@
     )
     --res
 
+// TODO: Convert into `:struct`
+:class val StringPair
```

Comment:
In the long run I think I'd like to generalize this to `Pair(String, String)` or just add tuples to the language. But this is okay for now.